### PR TITLE
Daemon mode added

### PR DESCRIPTION
**Pull request by F4ever**

### CODE REVIEW

**Review by sergeyWh1te — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -140,6 +140,12 @@ Full variables list could be found [here](https://github.com/lidofinance/lido-or
 > docker run --env EXECUTION_CLIENT_URI={value} --env CONSENSUS_CLIENT_URI={value} --env KEYS_API_URI={value} --env LIDO_LOCATOR_ADDRESS={value} lidofinance/oracle:{tag} {type}
 > ```
 
+## Manual mode
+
+Oracle could be executed once in "manual" mode. To do this setup `DAEMON` variable to 'True'.
```
Comment (skhomuti) [inline README.md:145]:
```suggestion
Oracle could be executed once in "manual" mode. To do this setup `DAEMON` variable to 'False'.
```

And I think it should be a more verbose paragraph about why we need this

File: src/web3py/extensions/tx_utils.py [hunk 2]
```diff
@@ -19,15 +20,27 @@ def check_and_send_transaction(self, transaction, account: Optional[LocalAccount
...
 
         return None
 
+    def _manual_tx_processing(self, transaction, params: TxParams, account: LocalAccount):
+        logger.warning({'msg': 'Send transaction in manual mode.'})
+        msg = (
+            '\n'
+            'Going to send transaction to blockchain: \n'
+            f'Tx args: {transaction.args}. \n'
+            f'Tx params: {params}. \n'
```
Comment (skhomuti) [inline src/web3py/extensions/tx_utils.py:39]: it may be a good idea to prettify json-like args

File: tests/web3_extentions/test_tx_utils.py [hunk 3]
```diff
@@ -72,3 +75,27 @@ def test_get_tx_params(web3, tx_utils, tx, account):
...
+def test_manual_tx_processing_decline(web3, tx_utils, tx, account):
+    input.get_input = Mock(return_value='n')
+    web3.transaction._sign_and_send_transaction = Mock()
+    web3.transaction._manual_tx_processing(tx, {}, account)
+    web3.transaction._sign_and_send_transaction.assert_not_called()
+
+
+def test_daemon_check_and_send_transaction(web3, tx_utils, tx, account):
+    input.get_input = Mock(return_value='n')
+    variables.DAEMON = False
```
Comment (skhomuti) [inline tests/web3_extentions/test_tx_utils.py:96]: do it like this:
```
    with monkeypatch.context():
        monkeypatch.setattr(variables, "DAEMON", False)
```
to not affect other tests

**Review by skhomuti — CHANGES_REQUESTED** [review]

File: src/variables.py [hunk 4]
```diff
@@ -39,9 +39,18 @@
 
 # Default delay for default Oracle members. Member with submit data role should submit data first.
```
[the remark this answers is not in the document]
Comment (F4ever) [reply src/variables.py:40]: fixed

File: README.md (hunk 1 above)
Comment (F4ever) [reply README.md]: Done

Comment (skhomuti) [reply README.md]: `docker run -ti --env-file .env --rm lidofinance/oracle:{tag} {type}` add command example

**Review by skhomuti — APPROVED** [review]

**Pull request merged**